LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -64,16 +64,10 @@ public class LabClient implements ILabClient {
      * Temp users API provided by Lab team can often take more than 10 seconds to return...hence, we
      * are overriding the read timeout.
      */
-    private static final int TEMP_USER_API_READ_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(15);
+    private static final int TEMP_USER_API_READ_TIMEOUT = (int) TimeUnit.SECONDS.toMillis(35);
 
     public static final long TEMP_USER_WAIT_TIME = TimeUnit.SECONDS.toMillis(35);
 
-    public static final String CREATE_TEMP_USER_API_CODE_SECRET_NAME = "FunctionApiCreateTempUserCode";
-    public static final String RESET_API_CODE_SECRET_NAME = "FunctionApiResetCode";
-    public static final String ENABLE_POLICY_API_CODE_SECRET_NAME = "FunctionApiEnablePolicyCode";
-    public static final String DISABLE_POLICY_API_CODE_SECRET_NAME = "FunctionApiDisablePolicyCode";
-    public static final String DELETE_DEVICE_API_CODE_SECRET_NAME = "FunctionApiDeleteDeviceCode";
```

Comment:
Are we moving from the functionApi back to the old way?

Reply:
Yes, the old implementation will interface with the lab api, which will route to calling the new functions that have been created to handle the methods above.